documentation/book/ref-key-features.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// assembly-overview.adoc
+
+[id='key-features-{context}']
+= Key Features
```

Comment:
Darn, can you give me a list of Strimzi Key Features? Just a few?

Reply:
Now you got me :-o. Never thought about it this way, because Strimzi's main mission is to enable Kafka on OpenShift. That is why I don't think it is wrong to list the Kafka features. We should just make it clear that these are Kafka features.

If I should put something together, it would be probably something like this:

Strimzi is

* Open source
* Easy to use
* Secure
* Flexible
* Kubernetes native

to deploy and manage Apache Kafka on Kubernetes and OpenShift.

Reply:
Alright, I'll go ahead and leave it as Kafka features for now :) We can revisit.